### PR TITLE
[doc] Enhances XSD documentation

### PR DESCRIPTION
**Pull request by reinhapa**
Added all avaliable attribute documentation from the example XML conf file to the XSD to be available within a schema based editor.

### CODE REVIEW

**Review by dizzzz — APPROVED** [review]
can't we generate documentation out of the schema?

File: schema/conf.xsd [hunk 1]
```diff
@@ -43,7 +55,13 @@
                                                             <xs:sequence>
                                                                 <xs:element ref="parameter" minOccurs="0" maxOccurs="unbounded"/>
                                                             </xs:sequence>
-                                                            <xs:attribute name="class" type="xs:string"/>
+                                                            <xs:attribute name="class" type="xs:string">
+                                                                <xs:annotation>
+                                                                    <xs:documentation>
+                                                                        The full class name ot the trigger implementation
```
Comment (adamretter) [inline schema/conf.xsd:61]: `ot` -> `of`

**Review by adamretter — COMMENTED** [review]
LGTM excluding one typo

File: schema/conf.xsd (hunk 1 above)
Comment (reinhapa) [reply schema/conf.xsd]: Done! :-)

**Comment by adamretter** [comment]
@reinhapa There is nothing wrong with this PR, and I am happy it was merged. However, just a polite reminder that submitters of PR's should never merge their own PR's please.

**Pull request merged**